sdk/java-sdk/src/test/scala/com/akkaserverless/javasdk/impl/MetadataImplSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk.impl
+
+import java.time.Instant
+
+import com.akkaserverless.javasdk.Metadata
+import com.akkaserverless.protocol.component.MetadataEntry
+import org.scalatest.OptionValues
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.jdk.OptionConverters._
+
+class MetadataImplSpec extends AnyWordSpec with Matchers with OptionValues {
+
+  "MetadataImpl" should {
+    "support getting the subject JWT claim" in {
+      metadata("_akkasls-jwt-claim-sub" -> "some-subject").jwtClaims.subject().toScala.value shouldBe "some-subject"
+    }
+
+    "support getting the expiriation JWT claim" in {
+      metadata("_akkasls-jwt-claim-exp" -> "12345").jwtClaims.expirationTime().toScala.value shouldBe Instant
+        .ofEpochSecond(12345)
+    }
+  }
+
```

Comment:
Would be good with some coverage of the additional parsing claim methods here.

Reply:
I've added more tests, particularly for the more complex parsers.